ui/src/pages/version/components/forms/components/DeploymentConfigPanel.js

```diff
@@ -138,6 +107,31 @@ export const EnvironmentPanel = ({
             disabled={isEnvironmentDisabled}
           />
         </EuiFormRow>
+        <EuiFormRow
+          fullWidth
+          label={
+            <EuiToolTip content="Specify the deployment mode of your model version">
+              <span>
+                Deployment Mode *{" "}
+                <EuiIcon type="questionInCircle" color="subdued" />
+              </span>
+            </EuiToolTip>
+          }
+          display="row">
+          <DeploymentModeDropdown
+            fullWidth
+            endpointStatus={endpoint.status}
+            deploymentMode={endpoint.deployment_mode}
+            onDeploymentModeChange={onDeploymentModeChange}
+          />
+        </EuiFormRow>
+        <EuiSpacer></EuiSpacer>
```

Comment:
I think it's better to move Min/Max replicas to this deployment panel instead of resources panel. Because autoscaling policy and min/max replicas should be near to each other.

Reply:
I think we shouldn't since there is also separate min/max replica for transformer. This panel is more intended for global configuration.